src/backend/aspen/api/views/groups.py

```diff
@@ -43,6 +43,8 @@ async def create_group(
     )
     organization = auth0_client.add_org(auth0_safe_prefix, group_creation_request.name)
     group_values = dict(group_creation_request) | {"auth0_org_id": organization["id"]}
+    if group_values.get("submitting_lab") is None:
+        group_values["submitting_lab"] = group_values["name"]
```

Comment:
i was thinking that we would have the group name be used as a default in the model field directly, something like (this from the sqlalchemy docs)[https://docs.sqlalchemy.org/en/14/core/defaults.html#context-sensitive-default-functions]: 

```
def mydefault(context):
    return context.get_current_parameters()['counter'] + 12

t = Table('mytable', metadata_obj,
    Column('counter', Integer),
    Column('counter_plus_twelve', Integer, default=mydefault, onupdate=mydefault)
)
```

Reply:
nice find!